### DISCUSSION TITLE
安装之后只能查到现货查不到合约

### DISCUSSION THREAD

**Discussion by huyubiao** [veighna-global/vnpy_evo#13] [Q&A]
![image](https://user-images.githubusercontent.com/54437582/187248429-a7206084-ffe9-4f40-96d1-c45d62fa6be6.png)
请教一下是不是只调了现货的接口？

**Comment by xldistance** [comment]
估计是要自己添加BinanceUsdtGateway接口，这个才是usdt永续接口
